client/my-sites/stats/activity-log/index.jsx

```diff
@@ -47,10 +56,37 @@ class ActivityLog extends Component {
 		translate: PropTypes.func.isRequired,
 	};
 
+	state = {
+		requestedRestoreTimestamp: null,
+		showRestoreConfirmDialog: false,
```

Comment:
Possibly we don't need `showRestoreConfirmDialog`, because it could be inferred from `requestedRestoreTimestamp`?